docs/systemd/polylith-example@.service

```
@@ -0,0 +1,14 @@
+[Unit]
+	Description= Dendrite Polylith Multi - %I
+	PartOf=polyDendrite.service
+	After=network.target
+
+	[Service]
+	User=dendrite
+	WorkingDirectory=/opt/dendrite
+	Type=forking
```

Should probably be `Type=simple`, since Dendrite doesn't fork a new process.
```suggestion
	Type=simple
```